test/fw/ptl/lib/pbs_testlib.py

```diff
@@ -14262,7 +14266,7 @@ class Reservation(ResourceResv):
 
     dflt_attributes = {}
 
-    def __init__(self, username=TEST_USER, attrs=None, hosts=None):
+    def __init__(self, username=TEST_USER, attrs=None, hosts=None, job=None):
```

Comment:
Instead of this, maybe key off of the reservation attribute instead of adding an argument to Reservation()?  That's how it will work in API mode.